lua/wikis/rematch/Infobox/League/Custom.lua

```diff
@@ -0,0 +1,57 @@
+---
+-- @Liquipedia
+-- page=Module:Infobox/League/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+
+local Injector = Lua.import('Module:Widget/Injector')
+local League = Lua.import('Module:Infobox/League')
+
+local Widgets = require('Module:Widget/All')
+local Cell = Widgets.Cell
+
+---@class RematchLeagueInfobox: InfoboxLeague
+local CustomLeague = Class.new(League)
+local CustomInjector = Class.new(Injector)
+
+-- Platform: Platform that the tournament is on
+local PLATFORMS = {
+	playstation = 'PlayStation',
+	xbox = 'Xbox',
+	steam = 'Steam',
+	cross = 'Cross-Platform',
+}
+
+---@param frame Frame
+---@return Html
+function CustomLeague.run(frame)
+	local league = CustomLeague(frame)
+	league:setWidgetInjector(CustomInjector(league))
+
+	league.args.platform = PLATFORMS[(league.args.platform or 'steam'):lower():gsub(' ', '')]
```

Comment:
not a fan of overwriting input
rather push it into self.data via function `CustomLeague:customParseArguments`

Reply:
needs testing (written on phone) but should work

Reply:
Does work fine (events with no platform input or ones not in the module will default to PC, the other input works normally too)